pennylane/gradients/vjp.py

```diff
@@ -136,7 +128,12 @@ def compute_vjp_single(dy, jac, num=None):
         if num == 1:
             jac = qml.math.squeeze(jac)
         jac = qml.math.reshape(jac, (-1, 1))
-        res = qml.math.tensordot(jac, dy_row, [[0], [0]])
+        try:

```

Comment:
I know for sure that tensorflow does not support `@`, do you know if that the only case where it would fail for the this one (compute_vjp_single)?